docs/coordinates/example_gallery_plot_galactocentric_frame.rst

```diff
@@ -0,0 +1,174 @@
+28.427958360720748 km / s 169.69916086104888 km / s 17.70831652451455 km / s
+
+It is sometimes useful to specify the solar motion using the
+`proper motion of Sgr A* <https://arxiv.org/abs/astro-ph/0408107>`_
+instead of Cartesian velocity components. With an assumed distance, we can convert
+proper motion components to Cartesian velocity components using `astropy.units`:
+
+>>> galcen_distance = 8 * u.kpc
+>>> pm_gal_sgrA = [-6.379, -0.202] * (u.mas / u.yr)  # from Reid & Brunthaler 2004
+>>> vy, vz = -(galcen_distance * pm_gal_sgrA).to(u.km / u.s, u.dimensionless_angles())
+
+We still have to assume a line-of-sight velocity for the Galactic center,
+which we will again take to be 11 km/s:
+
+>>> vx = 11.1 * (u.km / u.s)
+>>> v_sun2 = u.Quantity([vx, vy, vz])  # List of Quantity -> a single Quantity
+>>> gc_frame2 = coord.Galactocentric(
+...     galcen_distance=galcen_distance, galcen_v_sun=v_sun2, z_sun=0 * u.pc
+... )
+>>> gc3 = c1.transform_to(gc_frame2)
+>>> print(gc3.v_x, gc3.v_y, gc3.v_z)  # doctest: +FLOAT_CMP
+28.427958360720748 km / s 167.61484955608267 km / s 18.118916793584443 km / s
+
+The transformations also work in the opposite direction. This can be useful
+for transforming simulated or theoretical data to observable quantities. As
+an example, we will generate 4 theoretical circular orbits at different
+Galactocentric radii with the same circular velocity, and transform them to
+Heliocentric coordinates:
+
+.. plot::
+   :include-source:
+
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
+    >>> import astropy.coordinates as coord
+    >>> from astropy import units as u
+    >>> ring_distances = np.arange(10, 26, 5) * u.kpc
+    >>> circ_velocity = 220 * (u.km / u.s)
+    >>> phi_grid = np.linspace(90, 270, 512) * u.degree  # grid of azimuths
+    >>> ring_rep = coord.CylindricalRepresentation(
+    ...     rho=ring_distances[:, np.newaxis],
+    ...     phi=phi_grid[np.newaxis],
+    ...     z=np.zeros_like(ring_distances)[:, np.newaxis],
+    ... )
+    >>> angular_velocity = (-circ_velocity / ring_distances).to(
+    ...     u.mas / u.yr, u.dimensionless_angles()
+    ... )
+    >>> ring_dif = coord.CylindricalDifferential(
+    ...     d_rho=np.zeros(phi_grid.shape)[np.newaxis] * (u.km / u.s),
+    ...     d_phi=angular_velocity[:, np.newaxis],
+    ...     d_z=np.zeros(phi_grid.shape)[np.newaxis] * (u.km / u.s),
+    ... )
+    >>> ring_rep = ring_rep.with_differentials(ring_dif)
+    >>> gc_rings = coord.SkyCoord(ring_rep, frame=coord.Galactocentric)
+
+    First, let's visualize the geometry in Galactocentric coordinates. Here are
+    the positions and velocities of the rings; note that in the velocity plot,
+    the velocities of the 4 rings are identical and thus overlaid under the same
+    curve:
+
+    >>> fig, axes = plt.subplots(1, 2, figsize=(12, 6))
+    >>> _ = axes[0].plot(gc_rings.x.T, gc_rings.y.T, marker="None", linewidth=3)

```

Comment:
Why are all these underscores needed? As a reader I find them distracting.

Reply:
doctest complains about output otherwise because they return objects. In notebook, you could add `;` to suppress that but I am not sure about `>>>`. I find having to add so many `# doctest: +IGNORE_OUTPUT` tedious but that is another option. Other ideas welcome.

Reply:
This is user-facing documentation. The closer the examples are to what the users should write themselves the better. The `# doctest: +IGNORE_OUTPUT` instructions are not rendered in the HTML documentation, so they are a much better solution than the leading underscores.

Reply:
Done!